apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp

```diff
@@ -21,6 +21,8 @@
 #include <media/FileOutputDataSource.h>
 #include "tc_common.h"
 
+#define SECOND 2
```

Comment:
When you make any variable which can be used in many place, naming is very important.
I suggest RECORD_DURATION